tests/pode_tests/config.py

```diff
@@ -0,0 +1,13 @@
+from hypothesis.strategies import integers
+
+
+MAX_COORDINATE_EXPONENT = 15
+MAX_COORDINATE = 10 ** MAX_COORDINATE_EXPONENT
+MIN_COORDINATE = -MAX_COORDINATE
+MIN_CONTOUR_SIZE = 3
+MAX_CONTOUR_SIZE = 10
+MAX_HOLES_SIZE = 5
+MIN_REQUIREMENT = 0.1
+MIN_SITES_COUNT = 1
+MAX_SITES_COUNT = 5
+coordinates_strategies_factories = {int: integers}
```

Comment:
why are we not supporting `Fraction` geometries?

Reply:
Fractions and floats caused lots of bugs before in my tests. I've fixed quite a number of bugs since then, though, so I'll have to try them again.